src/main/java/put/apl/Algorithms/Sorting/Data/ConstantDataGenerator.java

```diff
@@ -0,0 +1,17 @@
+package put.apl.Algorithms.Sorting.Data;
+
+import org.springframework.stereotype.Component;
+
+import java.util.Arrays;
+import java.util.Random;
+
+@Component("constantData")
+public class ConstantDataGenerator implements  SortingDataGenerator{
+    @Override
+    public SortingData generate(DataGeneratorConfig config) {
```

Comment:
Czym to się różni od RandomData?

Reply:
Wypełnia całą listę jedną wylosowaną liczbą.